lesson_15/randycastro/Readme.js

```diff
@@ -0,0 +1,26 @@
+const http = require('http')
+const fs = require ('fs')
+const { error } = require('console')
+const port = 3000
+
+const server = http.createServer(function(req, res){
+res.writeHead(200, { 'Content-Type' : 'text/html'})
+fs.readFile ('readme.html', function (error, data) {
```

Comment:
You should use the information that I gave you in the articles to implement this in one line of code. Use express static middleware.